src/analyze/tableau/outlier_detection/outlier_detect_ilf.py

```diff
@@ -0,0 +1,101 @@
+import sys
+import argparse
+import json
+import pandas as pd
+from math import log
+from sklearn.ensemble import IsolationForest
+
+
+def read_file(f_name):
```

Comment:
Some comments needs to be added to this function what the purpose/outcome from this call, the function name is too generic and this call is doing little more that reading a file